src/lib/externaldata.py

```diff
@@ -40,8 +41,9 @@ class State(Enum):
         VALID = 1 << 1 # URL is reachable
         BROKEN = 1 << 2 # URL couldn't be reached
 
-    def __init__(self, data_type, filename, url, checksum, size=-1, arches=[],
-                 checker_data=None):
+    def __init__(self, data_type, pkg_name, filename, url, checksum, size=-1,
+                 arches=[], checker_data=None):
+        self.pkg_name = pkg_name
```

Comment:
See other comments regarding the `pkg_name`.

Reply:
Done